draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1592,13 +1592,13 @@ Each variant can be used as a MAC by calling the `Encrypt()` function with the m
 
 As shown in {{D23}}, AEGIS-128X and AEGIS-256X share the same security properties and requirements as AEGIS-128L and AEGIS-256 respectively. In particular, the security level and usage limits remain the same.
 
-Without the ability to set the associated data, a successful forgery doesn't increase the probability of subsequent forgeries. Reforgeability reilience of AEGIS is analyzed in {{FLLW17}}.
+Without the ability to set the associated data, a successful forgery does not increase the probability of subsequent forgeries and the reforgeability resilience of AEGIS is analyzed in {{FLLW17}}.
```

Comment:
Could we say something like `AEGIS has been shown to have reforgeability resilience {{FLLW17}}. Without the ability to...`? Or keep the first sentence but change the second sentence to `This property is known as reforgeability resilience` and do the citation the sentence before?

Reply:
I like the first option. Shorter sentences are indeed better. Changed, thanks!